cmd/generic-handlers.go

```diff
@@ -24,20 +24,17 @@ import (
 	"time"
 
 	humanize "github.com/dustin/go-humanize"
-	router "github.com/gorilla/mux"
 	"github.com/rs/cors"
 )
 
 // HandlerFunc - useful to chain different middleware http.Handler
 type HandlerFunc func(http.Handler) http.Handler
 
-func registerHandlers(mux *router.Router, handlerFns ...HandlerFunc) http.Handler {
-	var f http.Handler
-	f = mux
+func registerHandlers(h http.Handler, handlerFns ...HandlerFunc) http.Handler {
```

Comment:
This is fine because the semantics of registerHandlers does not require a router.Router. A Handler is sufficient.